source/StockMarket.py

```diff
@@ -0,0 +1,118 @@
+from Company import Company, valid_name
+
+# Stock Market Class exercise
+
+
+def unique_name(lst: list):
+    '''
+    A function which checks if the elements
+    in the list are unique
+    '''
+    if len(set(lst)) == len(lst):
+        return True
+    return False
+
+
+class StockMarket:
+
+    # The constructor function
+    def __init__(self,
+                 name: str,
+                 min_net_worth_threshold: int,
+                 stock_market_types: list,
+                 companies=[]):
+        if not valid_name(name):
+            raise ValueError("Error! Please enter a valid name.")
+        else:
+            self.name = name
+        if min_net_worth_threshold < 0:
+            raise ValueError("Error! Please enter an unnegative number")
+        else:
+            self.min_net_worth_threshold = min_net_worth_threshold
+        for ele in stock_market_types:
+            if not valid_name(ele):
+                raise ValueError("Error! Each element should be valid")
+        else:
+            self.stock_market_types = stock_market_types
```

Comment:
It seems like you have a problem with the tabs here

Reply:
I did not understand you point. What kind of problem?